frontend/src/homePage/Activities.jsx

```diff
@@ -0,0 +1,6 @@
+import "./Activities.css";
```

Comment:
Is this where group notifications will be shown?

Reply:
Yes, it is